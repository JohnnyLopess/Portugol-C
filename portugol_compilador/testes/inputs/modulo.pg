programa{
    funcao inicio() {
        se(5%2 == 1){
            escreva("o resto de 5 dividido por 2 é igual a 1")
        } senao {
            escreva("a matemática está errada")
        }
    }
}