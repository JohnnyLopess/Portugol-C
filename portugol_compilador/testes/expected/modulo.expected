#include <stdio.h>

int main() {
    if (5 % 2 == 1) {
        printf("o resto de 5 dividido por 2 é igual a 1");
    } else {
        printf("a matemática está errada");
    }
    return 0;
}